src/utils/child_process.ts

```diff
@@ -41,6 +41,15 @@ export async function execChildProcess(command: string): Promise<ExecChildProces
     return r;
 }
 
+export async function spawnChildProcess(command: string, onStdOut: (data: string) => void, onStdErr: (data: string) => void): Promise<number> {
+    const r: Promise<number> = new Promise((resolve, reject) => {
+        const process = child_process.spawn(command, []);
```

Comment:
This may require splitting the command into the executable path and the command line arguments. See the comment in  [`rustup.ts` at line 85](https://github.com/TomasHubelbauer/rls-vscode/blob/0a8d07219b86ca921512561486760743d5b0d6e0/src/rustup.ts#L85).